src/main/resources/exclusions.properties

```
@@ -68,6 +69,7 @@ resteasy-jsonb,spring-web=skip-all
 resteasy-multipart,spring-web=skip-all
 resteasy-reactive,oidc-client-filter=skip-all
 spring-web,oidc-client-filter=skip-all
+spring-web,rest-client-jsonb=skip-all
```

We should also add
```suggestion
spring-web,rest-client-jsonb=skip-all
spring-web,rest-client-jaxb=skip-all
spring-web,rest-client=skip-all
```
See failures in https://main-jenkins-csb-quarkusqe.apps.ocp-c1.prod.psi.redhat.com/job/quarkus-main-rhel8-jdk11-extensions-combinations-ts/117/.